.github/labeler.yml

```
@@ -229,6 +229,14 @@ team-infra:
     - any-glob-to-any-file:
       - docs/infra/**/*
 
+team-ios:
+  - changed-files:
+    - any-glob-to-any-file:
+      - engine/src/flutter/shell/platform/darwin/common/**/*
+      - engine/src/flutter/shell/platform/darwin/ios/framework/**/*
+      - packages/flutter_tools/**/ios/*
```

I would also do macos, which has a bunch of files with shared functionality
```suggestion
      - packages/flutter_tools/**/ios/*
      - packages/flutter_tools/**/macos/*
```